src/ResourceManagement/AppService/RuntimeStack.cs

```diff
@@ -7,6 +7,21 @@ namespace Microsoft.Azure.Management.AppService.Fluent
     /// </summary>
     public partial class RuntimeStack
     {
+        public static readonly RuntimeStack NodeJS_LTS = new RuntimeStack("NODE", "LTS");
+        public static readonly RuntimeStack NodeJS_10_LTS = new RuntimeStack("NODE", "10 LTS");
```

Comment:
I am not sure whether Azure support this version "10 LTS" (it is not on portal). Would you refer me to the source of the config lists?

Reply:
Seems should be "10-lts"? (result from "az webapp list-runtimes")

Reply:
Thanks for the good catch. I had a wrong reference list. 

Updated the list using ```az webapp list-runtimes --linux```. But now the question is, what about the Windows stack. It gives a different result? Is this class OS dependent?

Reply:
I think on current code design, withBuiltInImage(RuntimeStack) works for Linux only (it continues on flow after withExistingLinuxPlan or withNewLinuxPlan). Windows would initial a different flow, which continues as withPythonVersion etc.

Please give me some more time to confirm it and try out some test cases myself.

Thanks.